resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/constants.scala

```diff
@@ -94,6 +100,31 @@ package object constants {
   private[spark] val DEFAULT_SHUFFLE_MOUNT_NAME = "shuffle"
   private[spark] val INIT_CONTAINER_SECRET_VOLUME_NAME = "spark-init-secret"
 
+  // Hadoop Configuration
+  private[spark] val HADOOP_FILE_VOLUME = "hadoop-properties"
+  private[spark] val HADOOP_CONF_DIR_PATH = "/etc/hadoop/conf"
+  private[spark] val ENV_HADOOP_CONF_DIR = "HADOOP_CONF_DIR"
+  private[spark] val HADOOP_CONF_DIR_LOC = "spark.kubernetes.hadoop.conf.dir"
+  private[spark] val HADOOP_CONFIG_MAP_SPARK_CONF_NAME =
+    "spark.kubernetes.hadoop.executor.hadoopConfigMapName"
+
+  // Kerberos Configuration
+  private[spark] val HADOOP_KERBEROS_SECRET_NAME =
```

Comment:
The names here are not super clear. Can we rename `spark.kubernetes.kerberos.dt` to `spark.kubernetes.kerberos.delegationTokenSecretName` and `spark.kubernetes.kerberos.secretname` to `spark.kubernetes.kerberos.keyTabSecretName`, and rename the constants accordingly?

Reply:
done

Reply:
Can we also rename the constant val names?